### PR TITLE
The search endpoint now supports credit transfers

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

Allows the search endpoint to search transfers as well. All the normal search/filtering features should work on this!

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes, checking [accessibility](https://www.a11yproject.com/checklist/) etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver
- [ ] Add release notes to [unreleased changelog](https://github.com/teamsempo/SempoBlockchain/blob/master/CHANGELOG.md#unreleased)

### CODE REVIEW

**Comment by michielderoos** [comment]
[Test instance against 100,000 users and 1m transfers over one year](https://trello.com/c/9agKs4bH/492-test-instance-against-100000-users-and-1m-transfers-over-one-year)

**Comment by michielderoos** [comment]
[Add credit transfer to search endpoint](https://trello.com/c/SKomKNgC/849-add-credit-transfer-to-search-endpoint)

File: app/server/utils/search.py [hunk 1]
```diff
@@ -25,19 +30,33 @@ def generate_search_query(search_string, filters, order, sort_by_arg, include_us
...
+            'rank': 'rank',
+        }
     }
-    if sort_by_arg not in sort_types_to_database_types:
+
+    if sort_by_arg not in sort_types_to_database_types[search_type]:
         return {
-            'message': f'Invalid sort_by value {sort_by_arg}. Please use one of the following: {sort_types_to_database_types.keys()}'\
+            'message': f'Invalid sort_by value {sort_by_arg}. Please use one of the following: {sort_types_to_database_types[search_type].keys()}'\
         }
```
Comment (tristanhcole) [inline app/server/utils/search.py:60]: Paginate breaks this

**Review by tristanhcole — COMMENTED** [review]
Should this PR also update `credit_transfer_api.py` GET endpoint. The `/accounts` page actually hits the `/transfer_account` endpoint, not the `/search` endpoint, so wondering if we should match that or not?

Otherwise, looks good to me once that catch is fixed

**Comment by tristanhcole** [comment]
Closed by #726

**Pull request closed without merging**